app/views/world_locations/index.html.erb

```diff
@@ -4,12 +4,26 @@
 <div class="block">
   <div class="inner-block">
     <header class="page-header js-filter-list">
-      <h1 class="title">Worldwide</h1>
+      <%= render "govuk_publishing_components/components/title", {
```

Comment:
If we are using the proper title component, can we not remove all the whitehall level `h1` styling and leave it as the default `h1` styles instead?

**With whitehall styles**

<img width="1533" alt="Screenshot 2020-09-03 at 18 08 31" src="https://user-images.githubusercontent.com/7116819/92145745-9979b880-ee10-11ea-857d-8effeea2183f.png">


**With component styles** (just checked off the `#whitehall-wrapper .page-header h1 { }` rules in devtools)

<img width="1541" alt="Screenshot 2020-09-03 at 18 08 50" src="https://user-images.githubusercontent.com/7116819/92145757-9d0d3f80-ee10-11ea-8594-c8aba133ec90.png">


If it's not too much work, it might be good to look into how many instances of `.page-header` with a hard coded `h1` there are? 
If those instances can be easily replaced with the title component, we can remove those unsightly `#whitehall-wrapper .page-header h1 { }` rules. 🤷🏻‍♀️

Reply:
There are 202 instances of hard coded h1s 😬  I also don't yet have means to test everything thanks to AWS permissions. I agree that I'd like this to be fixed but are you happy to de-scope it for now?

Reply:
@owenatgov yes there are many hard coded `h1`s in Whitehall.
However the rules I highlighted only apply to hard coded `h1` inside a `page-header`, of which there are only 4 instances that I can see from doing a quick search.
It's definitely not within the scope of this PR though, so I will leave it to your discretion whether to do this or not.